frontend/src/components/IdeasList.jsx

```diff
@@ -38,10 +39,14 @@ const IdeasList = ({
     [base]
   );
 
-  const fetchUrl = getApiUrl(page);
   useEffect(() => {
+    const skip = page.number > 0 ? `&skip=${page.number * count}` : '';
+    const sorting = sort ? `&sort=${sort}` : '';
+
+    const fetchUrl = `${base}?limit=${count}${sorting}${skip}`;
+
     fetchFromApi(fetchUrl);
-  }, [fetchFromApi, fetchUrl]);
+  }, [fetchFromApi, base, count, sort, page]);
```

Comment:
This will cause constant requests to api. See home page. (Which was the reason for setting `fetchUrl` outside of the `useEffect` 😛 )

Reply:
Hey,

looks like this wasn't addressed yet. Could you take a look?

(I thought I added this comment before, but apparently I did not...)